src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -373,11 +367,11 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
     val expected = AppInfo(
       app.copy(
         versionInfo = AppDefinition.VersionInfo.OnlyVersion(clock.now()),
-        container = Some(app.container.get.copy(docker = Some(app.container.get.docker.get.copy(
+        container = Some(app.container.get.asInstanceOf[Container.Docker].copy(
```

Comment:
No `asInstanceOf`, please. I know it's only a test and the test case is about docker, but there is a type safe solution 😬